lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -56,15 +59,46 @@
 '''
 
 EXAMPLES = '''
-# docker_swarm.yml
+# Minimal example using local docker
+plugin: docker_swarm
+host: unix://var/run/docker.sock
+
+# Minimal example using remote docker
+plugin: docker_swarm
+host: tcp://my-docker-host:2375
+
+# Example using remote docker with unverified TLS
+plugin: docker_swarm
+host: tcp://my-docker-host:2375
+tls: yes
+
+# Example using remote docker with verified TLS and client certificate verification
+plugin: docker_swarm
+host: tcp://my-docker-host:2375
+tls_verify: yes
+cacert_path: /somewhere/ca.pem
+key_path: /somewhere/key.pem
+cert_path: /somewhere/cert.pem
+
+# Example using constructed features to create groups and set ansible_host
 plugin: docker_swarm
 host: tcp://my-docker-host:2375
+strict: False
+keyed_groups:
+  # add e.g. x86_64 hosts to an arch_x86_64 group
+  - prefix: arch
+    key: 'Description.Platform.Architecture'
+  # add e.g. linux hosts to an os_linux group
+  - prefix: os
+    key: 'Description.Platform.OS'
+  # create a group per node label
+  - key: 'Spec.Labels'
+    prefix: label
```

Comment:
Maybe add a comment/update the above comment that for example the label `production` will create a group `label_production`.

Reply:
Also, out of curiosity: have you (or @WojciechowskiPiotr) actually tried this (or a similar) example to see that the groups you claim to be created here are actually created?

Reply:
Jup,I've tried with some bogus labels to test the safe names